internal/suggestions/suggest.go

```diff
@@ -126,7 +126,7 @@ func Suggest(schema []byte, schemaPath string, errs []error, config Config) erro
 		suggest.Verbose = false
 
 		// Request suggestions in parallel, in batches of suggestionBatchSize
-		suggestions := make([]*Suggestion, len(errs))
+		var suggestions []*Suggestion
```

Comment:
this was/is a bug

Reply:
can we merge [this PR](https://github.com/speakeasy-api/speakeasy/pull/186) first or rebase on top of it